src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -982,7 +977,7 @@ internal struct UpdatedMemberInfo
                 // We seed the method body matching algorithm with tracking spans (unless they were deleted)
                 // to get precise matching.
                 TextSpan trackedSpan = default;
-                bool isTracked = trackingService?.TryGetSpan(new ActiveStatementId(documentId, ordinal), newText, out trackedSpan) ?? false;
+                bool isTracked = trackingServiceOpt?.TryGetSpan(new ActiveStatementId(documentId, ordinal), newText, out trackedSpan) ?? false;
```

Comment:
>trackingServiceOpt? [](start = 33, length = 19)

nit: maybe for consistency, it is worth to use the same approach for all such checks: either this one "trackingServiceOpt?... ?? false" or the approach above "trackingServiceOpt != null && ... "

Reply:
I'll follow up.